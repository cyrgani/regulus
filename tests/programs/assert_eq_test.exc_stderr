AssertionError: Equality assertion failed! lhs: `2`, rhs: `4`!
at <stl:assertions>:4:10
at <stl:assertions>:1:32
at programs/assert_eq_test.re:1:41
at programs/assert_eq_test.re:1:31

AssertionError: Equality assertion failed! lhs: `true`, rhs: `1`!
at <stl:assertions>:4:10
at <stl:assertions>:1:32
at programs/assert_eq_test.re:2:41
at programs/assert_eq_test.re:2:31

AssertionError: Equality assertion failed! lhs: `<function>(0)`, rhs: `1`!
at <stl:assertions>:4:10
at <stl:assertions>:1:32
at programs/assert_eq_test.re:3:41
at programs/assert_eq_test.re:3:31

AssertionError: Equality assertion failed! lhs: `{}`, rhs: `null`!
at <stl:assertions>:4:10
at <stl:assertions>:1:32
at programs/assert_eq_test.re:5:41
at programs/assert_eq_test.re:5:31

AssertionError: Equality assertion failed! lhs: `<function>(_)`, rhs: `<function>(_)`!
at <stl:assertions>:4:10
at <stl:assertions>:1:32
at programs/assert_eq_test.re:6:41
at programs/assert_eq_test.re:6:31

