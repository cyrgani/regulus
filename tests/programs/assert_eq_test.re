assert_eq(
    catch(assert_eq(2, 4)),
    "UserRaisedError: Equality assertion failed! lhs: `2`, rhs: `4`!"
),
assert_eq(
    catch(assert_eq(true, 1)),
    "UserRaisedError: Equality assertion failed! lhs: `true`, rhs: `1`!"
),

assert_eq(
    catch(assert_eq(fn(_()), 1)),
    "UserRaisedError: Equality assertion failed! lhs: `<function>(0)`, rhs: `1`!"
),
type(F),
assert_eq(
    catch(assert_eq(F(), null)),
    "UserRaisedError: Equality assertion failed! lhs: `{}`, rhs: `null`!"
),
assert_eq(
    catch(assert_eq(_, _)),
    "UserRaisedError: Equality assertion failed! lhs: `<function>(_)`, rhs: `<function>(_)`!"
),
