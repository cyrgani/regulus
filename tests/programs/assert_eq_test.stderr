AssertionError: Equality assertion failed! lhs: `2`, rhs: `4`!
at <stl:assertions>:4:10
at <stl:assertions>:1:32
at programs/assert_eq_test.re:1:32
at programs/assert_eq_test.re:1:22
AssertionError: Equality assertion failed! lhs: `true`, rhs: `1`!
at <stl:assertions>:4:10
at <stl:assertions>:1:32
at programs/assert_eq_test.re:2:32
at programs/assert_eq_test.re:2:22
AssertionError: Equality assertion failed! lhs: `<function>(0)`, rhs: `1`!
at <stl:assertions>:4:10
at <stl:assertions>:1:32
at programs/assert_eq_test.re:3:32
at programs/assert_eq_test.re:3:22
AssertionError: Equality assertion failed! lhs: `{}`, rhs: `null`!
at <stl:assertions>:4:10
at <stl:assertions>:1:32
at programs/assert_eq_test.re:5:32
at programs/assert_eq_test.re:5:22
AssertionError: Equality assertion failed! lhs: `<function>(_)`, rhs: `<function>(_)`!
at <stl:assertions>:4:10
at <stl:assertions>:1:32
at programs/assert_eq_test.re:6:32
at programs/assert_eq_test.re:6:22
